src/components/PlaylistAddModal.vue

```diff
@@ -87,7 +87,11 @@ export default {
                 if (json.error) alert(json.error);
             });
         },
-        async fetchPlaylists() {
+        async fetchPlaylists(createdPlaylistId) {
```

Comment:
We should probably move the logic when a new playlist got created to a new function that
1. takes two arguments: `playlistId` and `playlistName`
2. appends the created playlist to the `playlists` array
3. sets the `selectedPlaylistId` to the `playlistId` of the new playlist

Reply:
We don't actually need to do this API call here again when the user created a new playlist, since we have all info we need without any additional API request.

Reply:
Updated that locally for now, should I inline `fetchPlaylists` now that it's only used in `mounted`?

Reply:
Yes, sounds reasonable :+1:

Reply:
Done!